apps/widgps/settings.js

```diff
@@ -20,9 +20,13 @@ var mainmenu = {
   '' : {'title' : 'GPS widget'},
   '< Back' : back,
   "Cross icon" : {
-    value : !!settings.crossIcon ,
+    value : settings.crossIcon ,
```

Comment:
I guess in this case it's probably fine because default settings are loaded, but just in general please can we keep the `!!` in. It ensures that even if the settings get corrupted, the field is still turned into a boolean and so can be edited by the menu. If for instance somebody put `{crossIcon:"hello"}` in the settings file, users would be unable to change the setting if `!!` isn't there

Reply:
Good to know, thanks for the heads up!